### PR TITLE
[CBRD-22902] Create Jenkinsfile for 10.1 branch

### PR DESCRIPTION
**Pull request by sjkim777**
http://jira.cubrid.org/browse/CBRD-22902
http://jira.cubrid.org/browse/CUBRIDORG-144

This file is copied from Jenkinsfile of 10.2 and modified for 10.1 build env.
I tested it in my fork repository (sjkim777/cubrid ). 

**Verification** 
- No test fails.
- Build Artifacts are as below. 
  CUBRID-CCI-Windows-x64-10.1.3.7701-8d1f0f3.zip
  CUBRID-CCI-Windows-x86-10.1.3.7701-8d1f0f3.zip
  CUBRID-Windows-x64-10.1.3.7701-8d1f0f3.msi
  CUBRID-Windows-x64-10.1.3.7701-8d1f0f3.zip
  CUBRID-Windows-x86-10.1.3.7701-8d1f0f3.msi
  CUBRID-Windows-x86-10.1.3.7701-8d1f0f3.zip
  CUBRID-10.1.3.7702-b7d00f3f0-Linux.x86_64-debug.sh
  CUBRID-10.1.3.7702-b7d00f3f0-Linux.x86_64-debug.tar.gz
  CUBRID-10.1.3.7702-b7d00f3f0-Linux.x86_64.rpm
  CUBRID-10.1.3.7702-b7d00f3f0-Linux.x86_64.sh
  CUBRID-10.1.3.7702-b7d00f3f0-Linux.x86_64.tar.gz
  cubrid-10.1.3.7702-b7d00f3f0.tar.gz
  cubrid-10.1.3.7702-b7d00f3f0.zip
  CUBRID-CCI-10.1.3.7702-b7d00f3f0-Linux.x86_64-debug.tar.gz
  CUBRID-CCI-10.1.3.7702-b7d00f3f0-Linux.x86_64.tar.gz
  JDBC-10.1.3.7702-cubrid-src.jar
  JDBC-10.1.3.7702-cubrid.jar

- Please refer to http://ci.cubrid.org/view/all/job/test/ for details . And i will remove the test job after confirming that it is approved and working.

### CODE REVIEW

**Comment by cubridci** [comment]
Build comment file: 
--------------
**
** This Pull Request's title is '[CUBRIDORG-144] Create Jenkinsfile for CI.'.
** And it does not start with a related JIRA issue number **
** Please edit this Pull Request's title. (eg: '[CBRD-20309] Redesign vacuum data flush' **
**

--------------

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+              image 'cubridci/cubridci:10.1'
+              label 'linux'
+              alwaysPull true
+            }
+          }
+          environment {
+            MAKEFLAGS = '-j'
+          }
+          steps {
+            sh '/entrypoint.sh build'
```
[the remark this answers is not in the document]
Comment (sjkim777) [reply Jenkinsfile]: corrected as below
script {
              currentBuild.displayName = sh(returnStdout: true, script: './build.sh -v').trim()
          }
and modified build.sh to support '-v' option like develop branch.
Also i checked build display name on jenkins CI.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            
+            echo 'testing...'
```
[the remark this answers is not in the document]
Comment (sjkim777) [reply Jenkinsfile]: corrected it like develop branch.

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+              alwaysPull true
+            }
+          }
+          environment {
+            MAKEFLAGS = '-j'
+          }
+          steps {
+            sh '/entrypoint.sh build -m debug'
+            
+            echo 'testing...'
```
[the remark this answers is not in the document]
Comment (sjkim777) [reply Jenkinsfile]: corrected it like develop branch.

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+          agent {
+            node {
+              label 'windows'
+            }
+          }
+          steps {
+            echo 'Building 32bit...'
+            bat "win/build.bat build"
+
+            echo 'Packing...'
```
[the remark this answers is not in the document]
Comment (sjkim777) [reply Jenkinsfile]: corrected it as you commented.

File: Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,108 @@
+pipeline {
+  agent none
+
+  triggers {
+    pollSCM('H 3 * * *')
```
[the remark this answers is not in the document]
Comment (sjkim777) [reply Jenkinsfile]: removed triggers to run manually.

**Comment by sjkim777** [comment]
corrected as below. 
- Add Build display name and modify wrong comments.
- Add cubridmanagerserver as submodule.

checked cubridmanagerserver.  
- linux release/debug : bin/cub_manager exists. 
- window 32bit/64bit : bin/cub_manager.exe exists. 
- cubrid start & check message. 
"cubrid manager server start: success"

Please review again.

**Comment by sjkim777** [comment]
Should be merged for final verification. 
Please merge.

**Comment by xkyu** [comment]
You can merge the changes and resolve the issue yourself.

> Should be merged for final verification.
> Please merge.

**Pull request merged**